Workbooks/Communication Services/Overview/Overview.workbook

```diff
@@ -360,50 +364,125 @@
+                        "label": "OS version",
+                        "type": 2,
+                        "multiSelect": true,
+                        "quote": "'",
+                        "delimiter": ",",
+                        "query": "ACSCallSummary\r\n| where CallStartTime {time_range:query}\r\n| distinct OsVersion\r\n| order by OsVersion",
+                        "value": [
+                          "value::all"
+                        ],
+                        "typeSettings": {
+                          "additionalResourceOptions": [
+                            "value::all"
+                          ],
+                          "showDefault": false
+                        },
+                        "defaultValue": "value::all",
+                        "queryType": 0,
+                        "resourceType": "microsoft.communication/communicationservices"
                       }
                     ],
                     "style": "pills",
                     "queryType": 0,
-                    "resourceType": "Microsoft.Communication/CommunicationServices"
+                    "resourceType": "microsoft.communication/communicationservices"
                   },
                   "name": "voice-video-quality-params"
                 },
                 {
-                  "type": 3,
+                  "type": 12,
                   "content": {
-                    "version": "KqlItem/1.0",
-                    "query": "ACSCallDiagnostics\r\n| join kind=inner ACSCallSummary on CorrelationId, ParticipantId, EndpointId\r\n| where CallStartTime {time_range:query} and MediaType in ({call_quality_media_types}) and EndpointType in ({call_quality_endpoint_types})\r\n| order by CallStartTime asc, ParticipantStartTime asc\r\n| extend JitterQuality = iff(JitterAvg > 30, \"Poor\", \"Good\")\r\n| extend PacketLossRateQuality = iff(PacketLossRateAvg > 0.1, \"Poor\", \"Good\")\r\n| extend RoundTripTimeQuality = iff(RoundTripTimeAvg > 500, \"Poor\", \"Good\")\r\n| extend StreamQuality = iff((JitterQuality == \"Poor\") or (PacketLossRateQuality == \"Poor\") or (RoundTripTimeQuality == \"Poor\"), \"Poor\", \"Good\")\r\n| summarize count() by StreamQuality\r\n| render piechart",
-                    "size": 0,
-                    "title": "Stream quality",
-                    "queryType": 0,
-                    "resourceType": "Microsoft.Communication/CommunicationServices"
+                    "version": "NotebookGroup/1.0",
+                    "groupType": "editable",
+                    "title": "Stream health",
+                    "items": [
+                      {
+                        "type": 1,
+                        "content": {
+                          "json": "Streams are classified as healthy if they exhibit telemetry values within the following ranges:\r\n\r\n- Round trip time under 500 miliseconds\r\n- Packet loss rate of less than 10%\r\n- Jitter under 30 miliseconds"
```

Comment:
Typos   miliseconds => milliseconds (double 'l')

Reply:
You're right, fixed this too.